src/transformers/models/auto/modeling_auto.py

```diff
@@ -479,6 +486,7 @@
 MODEL_FOR_MULTIPLE_CHOICE_MAPPING_NAMES = OrderedDict(
     [
         # Model for Multiple Choice mapping
+("poolformer", "PoolFormerForMultipleChoice"),
```

Comment:
```suggestion
```
You can add `PoolFormerForImageClassification` to the MODEL_FOR_IMAGE_CLASSIFICATION_MAPPING_NAMES.